progression/src/main/java/me/mykindos/betterpvp/progression/profession/skill/woodcutting/BarkBounty.java

```diff
@@ -0,0 +1,139 @@
+package me.mykindos.betterpvp.progression.profession.skill.woodcutting;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import me.mykindos.betterpvp.core.listener.BPvPListener;
+import me.mykindos.betterpvp.progression.Progression;
+import me.mykindos.betterpvp.progression.profession.skill.ProgressionSkillDependency;
+import me.mykindos.betterpvp.progression.profession.woodcutting.WoodcuttingHandler;
+import me.mykindos.betterpvp.progression.profile.ProfessionProfileManager;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.Arrays;
+
+/**
+ * This perk grants a chance for the player to receive a <i>special item</i> when they strip a log.
+ * <p>
+ *     This item is <b>Tree Bark</b> which can sold at the shops
+ * </p>
+ */
+@Singleton
+@BPvPListener
+public class BarkBounty extends WoodcuttingProgressionSkill implements Listener {
+    ProfessionProfileManager professionProfileManager;
+    WoodcuttingHandler woodcuttingHandler;
+
+    /**
+     * Represents the percentage, per level, that <b>Tree Bark</b> will drop when any given log is stripped
+     */
+    private double barkChanceIncreasePerLvl;
+
+    /**
+     * Represents the log types that can be stripped in order to obtain <b>Tree Bark</b>
+     */
+    private final Material[] validLogTypes = new Material[]{
```

Comment:
Create a function in UtilBlock potentially, perhaps something like isLog(Material) to make it more future proof and less repeated